### ISSUE TITLE
a11y: problem with disabled state for radio buttons and checkboxes

### ISSUE THREAD

**Issue by aalves08** [rancher/dashboard#14252]
**Setup**
- Rancher version:
- Rancher UI Extensions:
- Browser type & version:

**Describe the bug**

`a11y issue 238`

The disabled state is not programmatically defined for mentioned form fields resulting in screen reader users are not able to understand the form field's disabled state.

Header Banner
Footer Banner
Login Screen Banner 
- Left, Center and Right radio buttons
- Bold, Italic and Underline checkboxes

**To Reproduce**

Go to the `banners` settings area

**Result**
The disabled state is not programmatically defined for mentioned form fields resulting in screen reader users are not able to understand the form field's disabled state.

**Expected Result**

The screen reader should announce the disabled state for the button.

**Screenshots**

**Additional context**

**Comment by yonasberhe23** [comment]
No longer able to use keyboard navigation on disabled radio groups and checkboxes. 

Tested on:

- Rancher v2.12-8f008ca4123bcb0ab521cc4a0a592d845db870c3-head
- Dashboard master `d269e5f`
